.github/workflows/pr-preview-links.yaml

```diff
@@ -0,0 +1,16 @@
+name: Read the Docs Pull Request Preview
+on:
+  pull_request_target:
+    types:
+      - opened
+
+permissions:
+  pull-requests: write
+
+jobs:
+  pr-preview-links:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: readthedocs/readthedocs-preview@main
+        with:
+          project-slug: "docs"
```

Comment:
I think next feature is a variable for .com/.org probably? Or perhaps using a URL for this, so we can just have a single var?

Reply:
I created an issue to track this at https://github.com/readthedocs/readthedocs-preview/issues/8